dotcom-rendering/src/types/editionsCrossword.ts

```diff
@@ -0,0 +1,43 @@
+export type FEEditionsCrosswords = {
+	quick: FEEditionsCrossword;
+	cryptic: FEEditionsCrossword;
+};
+
+type FECrosswordEntry = {
+	id: string;
+	number: number;
+	humanNumber: string;
+	direction: 'across' | 'down';
+	position: { x: number; y: number };
+	separatorLocations: {
+		','?: number[];
+		'-'?: number[];
+	};
+	length: number;
+	clue: string;
+	group: string[];
+	solution: string;
+	format: string;
+};
+
+type FECrosswordDimensions = {
+	cols: number;
+	rows: number;
+};
+
+export type FEEditionsCrossword = {
```

Comment:
Looks like this is based off this https://github.com/guardian/content-api/blob/509f9277510a423b5f9210737d7707ddf4656f16/porter/src/main/scala/com.gu.contentapi.porter/model/Crossword.scala

I thought it was going to be this model https://github.com/guardian/frontend/blob/3ed38eee41afee9f310ec6e0e0422a0917eca247/common/app/model/CrosswordData.scala#L177

Reply:
There's going to be a step in DCAR that transforms from one to the other, similar to how the enhancers work.

Reply:
But isn't the one coming from FE the same as the CAPI type?

Reply:
> But isn't the one coming from FE the same as the CAPI type?

That's the plan, yes. See https://github.com/guardian/frontend/pull/27604

Reply:
Which is this one -> 
 https://github.com/guardian/content-api/blob/509f9277510a423b5f9210737d7707ddf4656f16/porter/src/main/scala/com.gu.contentapi.porter/model/Crossword.scala

 without `annotatedSolution?: string;` and with optional `dateSolutionAvailable`

Reply:
The other one that is very similar but is not the same looks like it is for something called porter

Reply:
Took this offline. The final decision was that the transformation currently done in FE will be done in DCR now. We will continue working against that type for now and will expose a function from react-crossword to convert the CAPI data to the most desirable API for the crossword player itself